bin/build-spec-orient.sh

```diff
@@ -11,8 +14,8 @@ else
     diagram_dir=/orientd/$diagram_dir
 fi;
 
-mkdir -p $output_dir
-mkdir -p $diagram_dir
+mkdir -p $output_dir > /dev/null 2>&1 
```

Comment:
this is to prevent an annoying error message each time you run the script.

Reply:
Thank you!